client/build_code/webpack_common.js

```diff
@@ -58,9 +58,14 @@ const get_rules = ({ target_ie11, language, is_prod_build }) => {
       use: js_module_loader_rules,
       sideEffects: true,
     },
+    {
+      //NIVO_TODO: Temporary workaround for bug from Nivo. Will need to delete this as it gets fixed in newer versions
+      test: /react-spring/,
+      sideEffects: true,
```

Comment:
Seems good enough, please open an issue as a reminder about this once merged.

Reply:
Likely a minor difference, but the fix nivo maintainers applied to their own site was more granular, identifying specific `react-spring` modules with side-effects rather than flagging the whole thing as having side-effects. Their approach is likely a little more optimized for bundle size (but if this is just temporary, it's probably a minor enough difference to ignore).

See https://github.com/plouc/nivo/blob/master/scripts/patch-react-spring.js

Reply:
I changed it, but then just checked that the modules listed there are all of the modules in `@react-spring`